src/codegen/doc/mod.rs

```diff
@@ -870,6 +870,32 @@ where
                 )?;
             }
         }
+        // document function's Error
+        let gerror_out_parameter = no_array_length_params
+            .iter()
+            .filter(|param| {
```

Comment:
Please use find instead, it is basically the same except it will return the first found item.